iterations_runners/iterations_runner.java

```diff
@@ -181,20 +184,20 @@ public static void main(String args[]) throws
          *   - We print the result using toUnsignedString, thus interpreting
          *     the number as unsigned.
          */
-        long[] tsr_iter_times = new long[iterations];
-        Arrays.fill(tsr_iter_times, 0);
+        long[] cycleCounts = new long[iterations];
+        Arrays.fill(cycleCounts, 0);
 
         // Please, no reflection inside the timed code!
```

Comment:
The runner does use reflection to get the named benchmark:

```
        // reflectively call the benchmark's run_iter                           
        Class<?> cls = Class.forName(benchmark);                                
        java.lang.reflect.Constructor<?>[] constructors = cls.getDeclaredConstructors();
        assert constructors.length == 1;                                        
        Object instance = constructors[0].newInstance();                        
        BaseKrunEntry ke = (BaseKrunEntry) instance; // evil                    
```

That said, I do wonder of the worth of the comment. Maybe we should remove it.